test/emf_ecs/ecs_resources/ec2_launch/daemon/ecs_taskdef.tpl

```diff
@@ -0,0 +1,80 @@
+[
```

Comment:
This looks like a merge gone wrong. I don't think this `emf_ecs` folder should exist

Reply:
Removed the emf_ecs folder